components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -2010,6 +2019,10 @@ public Policy getSimplifiedPolicyByLevelAndName(APIMgtAdminService.PolicyLevel p
                             policy = new APIPolicy(resultSet.getString(APIMgtConstants
                                     .ThrottlePolicyConstants.COLUMN_UUID), resultSet.getString(APIMgtConstants
                                     .ThrottlePolicyConstants.COLUMN_NAME));
+                            setCommonPolicyDetails(policy, resultSet);
+                            ((APIPolicy) policy).setUserLevel(resultSet.getString(
```

Comment:
Any particular reason for doing a casting here?

Reply:
setUserLevel method is not defined in Policy class and only defined in APIPolicy class. Therefore, needed a casting before calling the method.